tests/jiva/clone/suite_test.go

```diff
@@ -60,6 +59,7 @@ func TestSource(t *testing.T) {
 
 func init() {
 	flag.StringVar(&kubeConfigPath, "kubeconfig", "", "path to kubeconfig to invoke kubernetes API calls")
+	flag.IntVar(&replicaCount, "replicas", 1, "value of replica count")
```

Comment:
can this be named as `jiva-replica-count`?
can this be placed at `tests/jiva/jiva.go` file

Reply:
If all three tests clone, volume and snapshot are running in parallel and we want different replicas for each tests. Will it work then ?

Reply:
How are they run in parallel?
Is `ginkgo -v ...` executed 3 times?
If yes, i guess every run can have their own path

Reply:
Yes as all the tests have their own suite we have to run `ginkgo -v ...` 3 times.

Reply:
Fine. Lets have a common flag for all test.

Reply:
Done